cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js

```diff
@@ -0,0 +1,90 @@
+import Permissions from '../../../../support/dictionary/permissions';
+import TopMenu from '../../../../support/fragments/topMenu';
+import BulkEditSearchPane from '../../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../../support/utils/fileManager';
+import Users from '../../../../support/fragments/users/users';
+import BulkEditActions from '../../../../support/fragments/bulk-edit/bulk-edit-actions';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+const editedFileName = `edited-records-${getRandomPostfix()}.csv`;
+const userUUIDsFileName = `userUUIDs_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('logs', () => {
+    before('Create test data', () => {
+      cy.createTempUser(
+        [
+          Permissions.uiUsersView.gui,
+          Permissions.bulkEditUpdateRecords.gui,
+          Permissions.uiUserEdit.gui,
+          Permissions.inventoryAll.gui,
+          Permissions.bulkEditView.gui,
+          Permissions.bulkEditEdit.gui,
+          Permissions.bulkEditCsvView.gui,
+          Permissions.bulkEditCsvEdit.gui,
+          Permissions.bulkEditLogsView.gui,
+        ],
+        'faculty',
+      ).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+        FileManager.createFile(`cypress/fixtures/${userBarcodesFileName}`, user.barcode);
+        FileManager.createFile(`cypress/fixtures/${userUUIDsFileName}`, user.userId);
```

Comment:
2) User has valid .csv files with Users and Items identifiers to upload for Bulk Edit 
https://foliotest.testrail.io/index.php?/cases/view/380546